src/php/Compiler/Domain/Emitter/OutputEmitter/NodeEmitter/MacroExpandEmitter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Compiler\Domain\Emitter\OutputEmitter\NodeEmitter;
+
+use Phel\Compiler\Domain\Analyzer\Ast\AbstractNode;
+use Phel\Compiler\Domain\Analyzer\Ast\MacroExpandNode;
+use Phel\Compiler\Domain\Emitter\OutputEmitter\NodeEmitterInterface;
+
+use function assert;
+
+final class MacroExpandEmitter implements NodeEmitterInterface
+{
+    use WithOutputEmitterTrait;
+
+    public function emit(AbstractNode $node): void
+    {
+        assert($node instanceof MacroExpandNode);
+
+        $form = $node->getValue();
+        $this->outputEmitter->emitNode($form);
```

Comment:
mmm and probably here, we might need to do something else to emit the macro code as PHP 🤔